src/main/java/bobby/Bobby.java

```diff
@@ -0,0 +1,112 @@
+                    tasks[entry - 1].setDone(false);
+                    System.out.println("Unmarked");
+                    System.out.println(entry + "." + tasks[entry - 1]);
+                }
+                break;
+            case "list":
+                for (int i = 0; i < counter; i += 1) {
+                    System.out.println((i + 1) + "." + tasks[i]);
+                }
+                break;
+            case "todo":
+                try {
+                    description = obtainTodoDescription(input);
+                } catch (BobbyException e) {
+                    System.out.println("Please enter a valid task.");
+                    break;
+                }
+                tasks[counter] = new Todo(description);
+                System.out.println("Okay, added:\n" + tasks[counter]);
+                counter += 1;
+                System.out.println("Now you have " + counter + " task(s) in the list.");
+                break;
+            case "deadline":
+                if (!input.contains("/by")) {
+                    System.out.println("Please enter a valid deadline.");
+                    break;
+                }
+                try {
+                    description = input.substring(9, input.indexOf("/by") - 1);
+                    by = input.substring(input.indexOf("/by") + 4);
+                } catch (StringIndexOutOfBoundsException e) {
+                    System.out.println("Please enter a valid deadline task in this format:");
+                    System.out.println("deadline Return Book /by Sunday");
+                    break;
+                }
+                tasks[counter] = new Deadline(description, by);
+                System.out.println("Okay, added:\n" + tasks[counter]);
+                counter += 1;
+                System.out.println("Now you have " + counter + " task(s) in the list.");
+                break;
+            case "event":
+                if (!input.contains("/to") || !input.contains("/from")) {
+                    System.out.println("Please enter a valid start and end date.");
+                    break;
+                }
+                try {
+                    description = input.substring(6, input.indexOf("/from") - 1);
+                    by = input.substring(input.indexOf("/to") + 4);
+                    from = input.substring(input.indexOf("/from") + 6, input.indexOf("/to") - 1);
+                } catch (StringIndexOutOfBoundsException e) {
+                    System.out.println("Please enter a valid event in this format:");
+                    System.out.println("event Project Meeting /from Mon 2pm /to 4pm");
+                    break;
+                }
+                tasks[counter] = new Event(description, by, from);
+                System.out.println("Okay, added:\n" + tasks[counter]);
+                counter += 1;
+                System.out.println("Now you have " + counter + " task(s) in the list.");
+                break;
```

Comment:
Perhaps abstraction could be applied here to shorten the methods. Same goes for the other cases in the switch statement.